app/src/lib/databases/pull-request-database.ts

```diff
@@ -6,7 +6,7 @@ export interface IPullRequestRef {
    * The database ID of the GitHub repository in which this ref lives. It could
    * be null if the repository was deleted on the site after the PR was opened.
    */
-  readonly repoId: number | null
+  readonly repositoryDbId: number | null
```

Comment:
I know this isn't ready for review yet but I wanted to make sure we either provide an upgrade function here that copies all `repoId` values over to `repositoryDbId` or clears the table to start over from scratch. See #3869 where this bit us before.

Reply:
Thanks for the heads up.